src/http/Stream.h

```diff
@@ -9,8 +9,14 @@
 #ifndef SQUID_SRC_HTTP_STREAM_H
 #define SQUID_SRC_HTTP_STREAM_H
 
+#include "clientStreamForward.h"
+#include "comm/forward.h"
+#include "debug/Stream.h"
```

Comment:
It would be better to stop inlining mayUseConnection(bool) instead, but I do not insist on that correction.

Reply:
I won't do it, this PR is already huge as it is